taxonomies/food/categories.txt

```diff
@@ -77029,14 +77047,32 @@ protected_name_file_number:en: PGI-DE-0224
 protected_name_type:en: pgi
 wikidata:en: Q1170667
 
-< en:German meat products
+< de:Bratwurst
+fr: Saucisses de Nuremberg, Nürnberger Rostbratwürstchen, Nürnberger Bratwürste, petites saucisses de Nuremberg
 bg: Нюрнбергски наденички
-de: Nürnberger Bratwürste, Nürnberger Rostbratwürste
+de: Nürnberger Bratwürste, Nürnberger Rostbratwürste, Original Nürnberger Rostbratwürste, Original Nürnberger Rostbratwurst
 origins:en: en:germany
 protected_name_file_number:en: PGI-DE-0184
 protected_name_type:en: pgi
 wikidata:en: Q11696257
 
+< de:Bratwurst
+de: Curry Bratwurst
+# This is the naked bratwurst seasonned with curry, not the currywurst full meal
+
+< en:German meat products
+< en:German sausages
+de: Käsekrainer
+wikidata:en: Q1571899
+
+< en:German meat products
```

Comment:
same

Reply:
I removed the "pdi type" en:German meat products and replaced by the "non-pdi type "german sausage". Is that correct ?